cmake/Modules/FindDPCPP.cmake

```diff
@@ -50,6 +50,7 @@ if(NOT "${DPCPP_SYCL_ARCH}" STREQUAL "")
   elseif("${DPCPP_SYCL_TARGET}" STREQUAL "nvptx64-nvidia-cuda")
     list(APPEND DPCPP_FLAGS "-Xsycl-target-backend")
     list(APPEND DPCPP_FLAGS "--cuda-gpu-arch=${DPCPP_SYCL_ARCH}")
+    list(APPEND DPCPP_FLAGS "-fsycl;-fsycl-targets=${DPCPP_SYCL_TARGET};-Xclang;-cl-mad-enable;-fsycl-unnamed-lambda")
```

Comment:
I don't think you need to repeat `-fsycl;-fsycl-targets=${DPCPP_SYCL_TARGET};`, which is already set in L45.
Probably, you shouldn't need `-fsycl-unnamed-lambda` as well. 

If needed, I suggest applying `-Xclang;-cl-mad-enable;` within the `if` in L70.